.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+    extends: "eslint:recommended",
+    parser: "babel-eslint",
+    parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: "module",
+        ecmaFeatures: {
+            jsx: true
+        }
+    },
+    env: {
+        browser: true,
+        es6: true
+    },
+    globals: {
+        process: "readonly"
+    },
+    plugins: ["react"],
+    rules: {
+        // Conflicted with Prettier
+        "no-extra-semi": "off",
+        "react/jsx-uses-react": "error",
+        "react/jsx-uses-vars": "error"
+    },
+    overrides: [
```

Comment:
Let's keep this contained to specific places. For a file, you can just add comment `/* eslint-env node */`. For tests create another `.eslintrc.js` which overrides env (see https://eslint.org/docs/user-guide/configuring#using-configuration-files-1).